### ISSUE TITLE
How to decode  opus?

### ISSUE THREAD

**Issue by zuowanbushiwo** [alexa/avs-device-sdk#1593]
**IMPORTANT**: Before you create an issue, please take a look at our [Issue Reporting Guide](https://github.com/alexa/avs-device-sdk/wiki/Issue-Reporting-Guide).

### What version of the AVS Device SDK are you using?
      <1.17.0.>

### Tell us what hardware you're using:
- [* ] Desktop / Laptop
- [ ] Raspberry Pi
- [ ] Other - tell us more:

### Tell us about your OS (Type & version):
- [*] Linux
- [ ] MacOS 
- [ ] Raspbian Stretch
- [ ] Raspbian Jessy
- [ ] Other - tell us more:

We sometimes need to analyze uploaded opus-encoded speech. I used libopus opus_demo and opus-tools opusdec to decode, but they were unsuccessful.
In other words, how to decode this Integration / inputs / utterance_time_success.opus file into PCM format? Thank you!

**Comment by womw** [comment]
Hi @zuowanbushiwo we currently do not provide any tools to help decode opus-encoded speech.

**Comment by zuowanbushiwo** [comment]
thanks!
